x-pack/plugins/uptime/public/components/overview/alerts/monitor_expressions/filters_expression_select.tsx

```diff
@@ -123,43 +133,63 @@ export const FiltersExpressionsSelect: React.FC<Props> = ({
     (curr) => curr.selectedItems.length > 0 || newFilters?.includes(curr.fieldName)
   );
 
+  const indexPattern = useIndexPattern();
+
   return (
     <>
-      {filtersToDisplay.map(({ description, value, ...item }) => (
-        <EuiFlexGroup key={item.id}>
-          <EuiFlexItem>
-            <FilterPopover
-              {...item}
-              btnContent={
-                <EuiExpression
-                  aria-label={item['aria-label']}
-                  color={'secondary'}
-                  data-test-subj={'uptimeCreateStatusAlert.' + item.id}
-                  description={description}
-                  value={value}
-                  onClick={() => setIsOpen({ ...isOpen, [item.id]: !isOpen[item.id] })}
+      {filtersToDisplay.map(
+        ({ description, id, title, value, fieldName, ariaLabel, selectedItems }) => (
+          <EuiFlexGroup key={id}>
+            <EuiFlexItem>
+              {indexPattern && (
+                <FieldValueSuggestions
+                  filters={[]}
+                  key={fieldName}
+                  indexPatternTitle={indexPattern.title}
+                  sourceField={fieldName}
+                  label={title}
+                  onChange={(vals) => {
+                    onFilterFieldChange(fieldName, vals);
+                  }}
+                  selectedValue={selectedItems}
+                  button={
+                    <EuiExpression
+                      aria-label={ariaLabel}
+                      color={'secondary'}
+                      data-test-subj={'uptimeCreateStatusAlert.' + id}
+                      description={description}
+                      value={value}
+                      onClick={() => setIsOpen({ ...isOpen, [id]: !isOpen[id] })}
+                    />
+                  }
+                  forceOpen={isOpen[id]}
+                  setForceOpen={() => {
+                    setIsOpen({ ...isOpen, [id]: !isOpen[id] });
+                  }}
+                  asCombobox={false}
+                  cardinalityField="monitor.id"
+                  time={{
+                    from: formatParamsDateRange(alertParams) ?? DEFAULT_FROM,
+                    to: to ?? 'now',
```

Comment:
Should this just always be `now`, since that's what the alert will do? The user could change the selected date and give the user a different idea of what the alert will actually be querying, in terms of monitor counts or the values that show up to filter by.